### PR TITLE
fix: Hide delivery message for group conversation

### PR DESCRIPTION
**Pull request by przemvs**
## Description

Hide delivery message for group conversation. We want to display only for 1:1 conversation delivery state icon.

## Checklist

- [ ] PR has been self reviewed by the author;
- [ ] Hard-to-understand areas of the code have been commented;
- [ ] If it is a core feature, unit tests have been added;

### Important details for the reviewers

(Delete this section if unnecessary)

- use (x) data
- can be reviewed commit-by-commit
- be sure to look at ...

### CODE REVIEW

File: src/script/components/MessagesList/Message/ContentMessage/ContentMessage.tsx [hunk 1]
```diff
@@ -265,9 +267,11 @@ export const ContentMessageComponent = ({
...
         </div>
 
-        <div css={deliveredMessageIndicator}>
-          <DeliveredMessage isLastDeliveredMessage={isLastDeliveredMessage} />
-        </div>
+        {message.expectsReadConfirmation && (
+          <div css={deliveredMessageIndicator}>
+            <DeliveredMessage isLastDeliveredMessage={isLastDeliveredMessage} is1to1Conversation={is1to1} />
+          </div>
+        )}
```
Comment (PatrykBuniX) [inline src/script/components/MessagesList/Message/ContentMessage/ContentMessage.tsx]: Wouldn't it be easier if we just didn't render the component at all if it's `!is1to1`? Passing a prop and rendering `null` doesn't really make sense to me 🤔  Same with `isLastDeliveredMessage`

Comment (przemvs) [reply src/script/components/MessagesList/Message/ContentMessage/ContentMessage.tsx]: if we don't render this div, then we will have jumping content :(

Comment (przemvs) [reply src/script/components/MessagesList/Message/ContentMessage/ContentMessage.tsx]: Also we using this component in 2 places, so better to pass inside component 2 props, and then component is reusable.

Comment (PatrykBuniX) [reply src/script/components/MessagesList/Message/ContentMessage/ContentMessage.tsx]: But we can render the div anyway and just render the message conditionally

**Review by PatrykBuniX — APPROVED** [review]

**Pull request merged**